core-bundle/src/Resources/contao/templates/backend/be_confirm.html5

```diff
@@ -59,4 +59,4 @@
   </div>
 
 </body>
-</html>
\ No newline at end of file
+</html>
```

Comment:
Please revert these changes.

Reply:
just had to merge upstream and now it's gone 🙃